hyperledger/grid-docs
# Initial update for website landing page

This PR replace the previous landing page content with text that's more appropriate for a general (non-developer) audience.
    
This commit also adds a Splinter logo to the /assets directory so that it's available if desired. (It was used in a previous draft of this PR.)

**View the changes at** https://grid-test.sawtooth.me/ 

**Test locally:**

1. Run this in the root directory: `just run`

2. Wait until you see `grid-docs-jekyll       |   Server running... press ctrl-c to stop.`

3. Point your browser to `0.0.0.0:8080`

Signed-off-by: Anne Chenette <chenette@bitwise.io>
